doc/whats_new.rst

```diff
@@ -31,6 +31,8 @@ Changelog
 
     - Add support for the University of Maryland KIT system by `Christian Brodbeck`_
 
+    - Add the `--filterchpi` option to `mne browse_raw`, by `Felix Raimundo`_
```

Comment:
Makes sense ^^

I can look into this in a new commit, I guess that with some sphinx-fu it is possible to create a `:command:` like `:func:`